tf2onnx/tfonnx.py

```diff
@@ -1638,6 +1638,24 @@ def where_op(ctx, node, name, args):
     return [node, transpose_node]
 
 
+def round_op7(ctx, node, name, args):
+    dtype = ctx.get_dtype(node.input[0])
```

Comment:
add TF signature like we did for better understanding. 

"    # T output = Softmax(T logits). The axis softmax would be performed on is always on -1.
     # T output = Softmax(T input, @int axis). Default axis is 1."